templates/components/component_library.html

```diff
@@ -0,0 +1,67 @@
+{% extends "base.html" %}
+{% load humanize %}
+{% load guardian_tags %}
+{% load static %}
+{% load q %}
+
+{% block title %}
+{{ portfolio.title }} Components Selected
+{% endblock %}
+
+{% block head %}
+{% include "controls/_style-controls.html" %}
+
+{% endblock %}
+
+{% block contextbar %}{% endblock %}
+
+{% block body %}
+
+  <div class="row" style="">
+    <div id="above-tab-content" class="row">
+      <!-- Page Title-->
+      <div style="float: left;">
+          <h2 class="control-heading" style="">
+            <a href="{{ project.get_absolute_url }}" style="color: black;">Component Library
+          </h2>
+      </div>
+      <!-- Control Lookup-->
+      <div style="float: right;">
+          <!-- <form id="control-lookup" method="get" action="/controls/" onsubmit="show_control_by_id(); return false;">
+            <input type="text" name="id" placeholder="Enter control id" value="{% if control %}{{ control.id_display|upper }}{% endif %}">
+                <button type="submit" class="btn btn-success">Look up</button>
+          </form> -->
+      </div>
```

Comment:
nitpick: Can probably remove this.

Reply:
Agreed. I had been leaving commented code alone but will start removing it as I see it.

Reply:
Removing the commented-out control search code from `templates/components/component_library.html` and will push change.